src/jit/codegencommon.cpp

```diff
@@ -4019,6 +4019,9 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
                 }
                 else
                 {
+                    // Currently the only non-HFA multireg struct is on ARM64 
+                    // and is two registers in size (i.e. two slots)
+                    assert(varDsc->lvSize() == (2 * TARGET_POINTER_SIZE));
```

Comment:
This comment is not correct. We also have non-Hfa multireg structs on amd64/ux, and we have longs on 32-bit architectures. As it happens, they are always two registers. But the comment is misleading.